src/olympia/abuse/models.py

```diff
@@ -785,6 +785,24 @@ def target(self):
         else:
             return self.addon
 
+    @property
+    def is_reportable(self):
+        """Is this abuse report reportable under DSA, so should be sent to Cinder"""
+        return bool(
+            self.reason in AbuseReport.REASONS.REPORTABLE_REASONS
+            and (
+                not self.guid
+                or (
+                    Addon.unfiltered.filter(guid=self.guid).exists()
+                    and not Version.unfiltered.filter(
+                        addon__guid=self.guid,
+                        version=self.addon_version,
+                        channel=amo.CHANNEL_UNLISTED,
+                    ).exists()
```

Comment:
This should be done the opposite way, looking for the version that exists in the listed channel. That way:
- If we ever add more channels, it will be an explicit decision to make them reportable (we are more likely to add channels that don't matter under the DSA than the opposite)
- Non-existing versions that are somehow reported by users won't be sent to Cinder (nothing can be done about them, since they don't exist, so they shouldn't be sent to Cinder)

(Also adjust tests accordingly)

Reply:
I wrote it deliberately that way as that's what @abyrne-moz specified in the issue.  I don't disagree with your logic though.

Reply:
I think inverting the logic like I'm suggesting also satisfies the criteria specified in the issue while having the advantages I've highlighted

Reply:
It does, but it opens more cases.  Such as reporting an add-on without addon_version supplied (i.e. what happens if you don't have the add-on installed at the time you report it, or use another browser) - should those cases be treated as if you reported a listed version?  e.g. Should we add another query to verify there is at least one listed version?

Reply:
https://github.com/mozilla/addons-server/pull/22625/commits/88243020ed2247222c29ed4079a9500baa79fbd8